torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +13,47 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+def nll_loss(g, self, target, weight, reduction, ignore_index):
+    # none reduction : onnx::Constant[value={0}]
+    # mean reduction : onnx::Constant[value={1}]
+    # sum reduction : onnx::Constant[value={2}]
+    reduction = sym_help._maybe_get_const(reduction, 'i')
+    reduction_vals = ['none', 'mean', 'sum']
+    reduction = reduction_vals[reduction]
```

Comment:
why is reduction integer? should it not be string?

Reply:
nll_loss receives an int : (0 for reduction, 1 for mean and 2 for sum).
The above 3 lines are to convert the received int value to the corresponding string.